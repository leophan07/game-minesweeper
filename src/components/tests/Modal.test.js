import React from 'react';
import { render } from '@testing-library/react';

import { Modal, ModalContent } from '../Modal';

describe('<App />', () => {

  it('Modal should render an <div> tag', () => {
    const { container } = render(<Modal />);
    expect(container.querySelector('div')).toBeTruthy();
  });

  it('ModalContent should render an <h1> tag', () => {
    const { container } = render(<ModalContent />);
    expect(container.querySelector('div')).toBeTruthy();
  });

  it('Modal should match the snapshot', () => {
    const { container } = render(
      <Modal>
        <ModalContent />
      </Modal>
    );
    expect(container).toMatchSnapshot();
  });
});
