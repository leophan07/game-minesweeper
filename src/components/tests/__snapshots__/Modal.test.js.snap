// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> Modal should match the snapshot 1`] = `
<div>
  <div
    class="sc-bdVaJa eONkMp"
  >
    <div
      class="sc-bwzfXH gXLqpO"
    />
  </div>
</div>
`;
